File: Backend/App/models/followup.py
from sqlalchemy import Column,String,Integer,Boolean,ForeignKey,DateTime
from Backend.App.core.database import Base
from Backend.App.models.prescription import Prescription

class FollowUp(Base):
    __tablename__ ="FollowUp"

    FollowUpId = Column(Integer,primary_key=True,index=True)
    PrescriptionId=Column(Integer,ForeignKey(Prescription.PrescriptionId))
    Date=Column(DateTime)
    Description = Column(String(255))

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/FollowUp.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class FollowUp
    {
        [Key]
        public int FollowUpId { get; set; }
        [Required]
        public int PrescriptionId { get; set; }

        [ForeignKey(nameof(PrescriptionId))]
        public Prescription Prescription { get; set; }
        public string Description { get; set; }
        public DateOnly FollowUpDate { get; set; }
    }
}
